### ISSUE TITLE
[Table | a11y]: Multiselect table with demandpopin is having color contrast issues

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents#10022]
### Bug Description

Table component in Multiselect mode is having color contracts issue when a row is selected with demand-popin property set
![image](https://github.com/user-attachments/assets/fd98e0ba-99b1-40a1-a5d9-aaefbc6322af)

### Affected Component

Table

### Expected Behaviour

There should be no color contrast issue

### Steps to Reproduce

1. Navigate to https://stackblitz.com/edit/js-3j1lsc?file=index.js,package.json,index.html,style.css
2. select any row and check the contract issues for popin column labels

### Priority

None

### UI5 Web Components Version

1.24.8

### Browser

Chrome

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by PetyaMarkovaBogdanova** [comment]
HI colleagues from @SAP/ui5-webcomponents-topic-tbl,
can you, please, check if there is an actual contrast ratio issue in selected table row?
Best regards,
Petya Markova.
(UI5 Dispatcher)

**Comment by DonkeyCo** [comment]
Hi @SAP/ui5-webcomponents-core,

this issue is regarding the V1 Table. Could you please have a look from your side? Seems like the colors may be incorrect here.

Best regards,
Duc

**Comment by nnaydenow** [comment]
Hi @lateefsofi,

The contrast ratio for the sap_horizon theme appears to be within acceptable norms.

As per the guidelines:

Text and icons must maintain a contrast ratio of at least 4.5:1 between the foreground and background colors in the default theme.
In high-contrast themes, the minimum contrast ratio should be 7:1.
Based on this, I’m closing the incident.

Best regards,
Nayden
